### PR TITLE
Added github workflow to test/build/publish wheel

### PR DESCRIPTION
**Pull request by ader1990**
GitHub workflows to run unit tests, install and build wheels for PyMI using Python 3.6

### CODE REVIEW

File: .github/workflows/pymi-cd.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: windows-latest
+    strategy:
+      max-parallel: 100
+      matrix:
+        python-version: [3.6]
```
Comment (petrutlucian94) [inline .github/workflows/pymi-cd.yml]: Let's also add 3.7 and 3.8.

File: README.rst [hunk 2]
```diff
@@ -11,3 +11,5 @@ PyMI
 Windows Management Infrastructure API for Python.
 
 For more details read the project's `README.rst <PyMI/README.rst>`_.
+
+rerun
```
Comment (petrutlucian94) [inline README.rst]: I think this was added by mistake.

Comment (ader1990) [reply README.rst]: yeap, will remove.

Comment (petrutlucian94) [reply README.rst]: I've just realized that Python 3.9 was released as well in the meantime. We should add that to the list as well.

**Pull request merged**